src/porepy/applications/verification_setups/manu_poromech_nofrac.py

```diff
@@ -595,49 +717,71 @@ def body_force(self, subdomains: list[pp.Grid]) -> pp.ad.Operator:
 
 
 class ModifiedEquationsPoromechanics(ModifiedMassBalance, ModifiedMomentumBalance):
-    ...
+    """Mixer class for modified poromoechanics equations."""
 
 
-# ---------> Solution Strategy
 class ModifiedSolutionStrategy(poromechanics.SolutionStrategyPoromechanics):
     """Solution strategy for the verification setup."""
 
+    darcy_flux: Callable[[list[pp.Grid]], pp.ad.Operator]
+    """Method that returns the Darcy fluxes in the form of an Ad operator. Usually
+    provided by the mixin class :class:`porepy.models.constitutive_laws.DarcysLaw`.
+
+    """
+
+    exact_sol: ManuPoroMechExactSolution
+    """Exact solution object."""
+
     fluid: pp.FluidConstants
+    """Object containing the fluid constants."""
+
     mdg: pp.MixedDimensionalGrid
-    darcy_flux: Callable[[list[pp.Grid]], pp.ad.Operator]
+    """Mixed-dimensional grid."""
+
+    plot_results: Callable
+    """Method for plotting results. Usually provided by the mixin class
+    :class:`SetupUtilities`.
+
+    """
+
+    results: list[SaveData]
+    """List of SaveData objects."""
+
     stress: Callable[[list[pp.Grid]], pp.ad.Operator]
-    results: list[StoreResults]
-    exact_sol: ExactSolution
+    """Method that returns the (integrated) poroelastic stress in the form of an Ad
+    operator. Usually provided by the mixin class
+    :class:`porepy.models.poromechanics.ConstitutiveLawsPoromechanics`.
 
-    def __init__(self, params: dict):
-        """Constructor for the class"""
+    """
 
+    def __init__(self, params: dict):
+        """Constructor for the class."""
         super().__init__(params)
 
-        # Exact solution
-        self.exact_sol: ExactSolution
-        """Exact solution object"""
+        self.exact_sol: ManuPoroMechExactSolution
+        """Exact solution object."""
 
-        # Prepare object to store solutions after each `stored_time`.
-        self.results: list[StoreResults] = []
-        """Object that stores exact and approximated solutions and L2 errors"""
+        self.results: list[SaveData] = []
+        """Results object that stores exact and approximated solutions and errors."""
 
-    def set_materials(self):
-        """Set material parameters.
+        self.flux_variable: str = "darcy_flux"
+        """Keyword to access the Darcy fluxes."""
 
-        Add exact solution object to the simulation model after materials have been set.
-        """
+        self.stress_variable: str = "poroelastic_force"
```

Comment:
I don't like adding "variables" which are not primary and then storing them in STATE. This obscures what are really primary variables imo.

Reply:
I don't particularly think that it will obscure what are the primary variables. After all, the user should be minimally familiar with MPFA/MPSA methods and know that the primary variables are pressure and displacement. Even if that's not the case, it should evident from the verification that these variables were added in a post-processing stage.

That being said, I can add a comment explaining that these are not primary variables. What do you think?

Reply:
I went for your approach. Fixed.